server/views/referrals/checkAnswers.njk

```diff
@@ -3,10 +3,9 @@
 
 {% extends "../partials/layout.njk" %}
 
-{% set pageTitle = "HMPPS Interventions" %}
+{% set pageTitle = "HMPPS Interventions - Make a referral - Check your answers" %}
 {% block pageTitle %}
-  {{ pageTitle }}
-  - GOV.UK
+  HMPPS Interventions - {{ pageTitle }} - GOV.UK
```

Comment:
I think that the prefix and suffix will be duplicated here?

Reply:
good spot, fixed